docs/kyma/docs/04-04-gke-installation.md

```diff
@@ -7,13 +7,12 @@ This Installation guide shows developers how to quickly deploy Kyma on a [Google
 By default, Kyma is installed on a GKE cluster with a wildcard DNS provided by [xip.io](http://xip.io). Alternatively, you can provide your own domain for the cluster. 
 
 ## Prerequisites
-- [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project
-- [Kubernetes](https://kubernetes.io/) 1.12
-- Tiller 2.10.0 or higher
```

Comment:
Why are we removing information about the required Tiller version?

Reply:
cause it is not a prerequisite, you install tiller on GKE cluster later on as part of instruction